toc/rfc/rfc-draft-bindable-secrets.md

```diff
@@ -0,0 +1,292 @@
+# Meta
+[meta]: #meta
+- Name: Bindable Secrets
+- Start Date: 2024-10-08
+- Author(s): @tcdowney @gerg
+- Status: Draft
+- RFC Pull Request: https://github.com/cloudfoundry/community/pull/994
+
+## Summary
+
+**tl;dr** Kubernetes-style Secrets for Cloud Foundry
+
+This RFC proposes adding a `secret` resource to the V3 Cloud Foundry API, similar in design and purpose to the Kubernetes [Secret resource](https://kubernetes.io/docs/concepts/configuration/secret/). Cloud Foundry Secrets will store arbitrary data in [CredHub](https://docs.cloudfoundry.org/credhub/) inject it into app containers via `tmpfs`-mounted files at a given path, using the same mechanism as [RFC 0030 - Add Support for File based Service Binding Information](https://github.com/cloudfoundry/community/blob/main/toc/rfc/rfc-0030-add-support-for-file-based-service-binding.md).
```

Comment:
That's good to know. I was under the impression that CredHub was a required CF component at this point since it had been a default cf-deployment component for so long. Out of curiosity why don't y'all use it? I feel it has a much stronger security posture than Cloud Controller.

I can rework this to be CredHub optional of course.

Reply:
I took a pass at making CredHub optional: 274b3b0